src/gui/ar/areaTargetScanner.js

```diff
@@ -115,18 +115,19 @@ createNameSpace("realityEditor.gui.ar.areaTargetScanner");
 
         realityEditor.app.onAreaTargetGenerateProgress('realityEditor.gui.ar.areaTargetScanner.onAreaTargetGenerateProgress');
 
-        realityEditor.app.subscribeToAppMemoryEvents('realityEditor.gui.ar.areaTargetScanner.onAppMemoryEvent');
-
-        realityEditor.gui.settings.addToggleWithText('Limit Scan RAM', 'area target scan stops at threshold (e.g. 0.33)', 'maximumRAM', '../../../svg/powerSave.svg', false, '0.33',
-            function(newValue) {
-                console.log('limitScanRAM was set to ' + newValue);
-                limitScanRAM = newValue;
-            },
-            function(newValue) {
-                console.log('zone text was set to ' + newValue);
-                maximumPercentRAM = parseFloat(newValue) || 0.33;
-            }
-        ).moveToDevelopMenu();
+        // realityEditor.app.subscribeToAppMemoryEvents('realityEditor.gui.ar.areaTargetScanner.onAppMemoryEvent');
```

Comment:
Shouldn't we just delete this rather than comment it out?

Reply:
Good point! Updated accordingly